### DISCUSSION TITLE
GUI to Code Mapping

### DISCUSSION THREAD

**Discussion by kamaaina** [vial-kb/vial-gui#212] [Q&A] [answered]
In the GUI, if you look at QMK Settings -> Tap-Hold tab, there is a checkbox for Ignore Mod Tap Interrupt. What is that mapped to in the QMK firmware code? I want to be able to set this flag by default when I compile my firmware.

I tried setting this in config.h, but got an error when i tried to compile.
`#define IGNORE_MOD_TAP_INTERRUPT`

error was:
`IGNORE_MOD_TAP_INTERRUPT is no longer necessary as it is now the default behavior of mod-tap keys. Please remove it from your config.`

After removing that define from config.h and re-flashing firmware, I opened the GUI and that checkbox is not checked. The behavior I am expecting is also incorrect. When I check the checkbox and click Save button, I get the desired behavior.

I'm trying to figure out what code modification I need to make to have this setting enabled by default.

Thank you!

**Comment by xyzz** [comment, marked as answer]
There isn't a configuration setting to change the default values for tap-hold behavior (although perhaps the default should follow qmk's), but you can patch it in https://github.com/vial-kb/vial-qmk/blob/f06a05a5d5e2e4e92d4ae28628cebaba2b9c32f0/quantum/qmk_settings.c#L184 by changing it to `QS.tapping = 2;`

**Reply by kamaaina** [reply]
thank you @xyzz !
